### ISSUE TITLE
Mojocast #3: Authentication, Helpers, and Plugins

### ISSUE THREAD

**Issue by tempire** [tempire/articles#11]
created_at=2011-09-12T16:21:00

The response for Mojocast #2 was even more overwhelming than the first episode.

How overwhelming?

<img class="rounded-corners" alt="mojocast e2 #1 on hackernews.png" src="http://blogs.perl.org/users/tempire/Screen%20Shot%202011-09-10%20at%208.05.49%20PM.png" style="margin: 20px 0 30px 10px" />

That's right, your upvotes matter.  Within 6 minutes of posting, it was secured on the front page.  Within 20 minutes, it was #1.  

Consider the reality here: the most watched site on the internet by hackers/developers had a Perl-related post in the top spot.  It remained on the front page for 6 hours, which is pretty good for a screencast with no article attached.  That's a solid group of folks with a lot of potential <strong><em>aha</em></strong> moments in Perl's favor.

Bottom line: If you care about the marketing of Perl and its exposure among the masses, consider upvoting the Mojocast, leave a comment, and spread the word!

Hooray Perl!

Speaking of which, 

<a href="http://mojocasts.com/e3">Mojocast #3: Authentication, Helpers, and Plugins</a>

<a href="http://hackerne.ws/item?id=2987819">Hackernews post</a>
